poetry.lock

```diff
@@ -4740,6 +4740,41 @@ docs = ["Pillow (>=7.1.2)", "matplotlib (>=2.2.3)", "memory-profiler (>=0.57.0)"
 examples = ["matplotlib (>=2.2.3)", "pandas (>=0.25.0)", "scikit-image (>=0.14.5)", "seaborn (>=0.9.0)"]
 tests = ["black (>=21.6b0)", "flake8 (>=3.8.2)", "matplotlib (>=2.2.3)", "mypy (>=0.770)", "pandas (>=0.25.0)", "pyamg (>=4.0.0)", "pytest (>=5.0.1)", "pytest-cov (>=2.9.0)", "scikit-image (>=0.14.5)"]
 
+[[package]]
```

Comment:
I can't understand why poetry is doing this with `scipy` and `statsmodels`...

Reply:
This happenned when I ran `poetry update scikit-learn`.

Reply:
The strange thing is that is added two versions of `scipy` here. But in a current `master` we have two versions of `statsmodels`.

Reply:
I have a suspicion that it uses two different versions for different python versions. I'll probably revert update in lock in this PR.

Reply:
Why do you run `poetry update scikit-learn`?